src/Packagist/WebBundle/Resources/views/layout.html.twig

```diff
@@ -204,6 +204,7 @@
 
         <script>
             var algoliaConfig = {{ algolia|json_encode|raw }};
+            algoliaConfig['tags'] = '{{ tags|default(false) }}';
```

Comment:
why not setting `tags` in the `algolia` structure before encoding it ?

Reply:
I agree it would be better but I'm not sure how to do it. Right now algolia is defined in the [config.yml](https://github.com/composer/packagist/blob/master/app/config/config.yml#L38-L41) as a twig global. Any advice?